Logging for Inserting & Removing Things Surgically

VickiMorris opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds logging for additional surgical processes, as currently with the three operations affected in this PR, it logs when each is started and when an organ is removed via Organ Manipulation. This makes it so organ manipulation logs when organs are inserted as well, cavity implant logs when items are put into or taken out of a body cavity surgically, and when implants are surgically removed from someone or they are checked for implants.

Adds logging for when any of the above procedures are *attempted* as well, encase the surgeon is interrupted mid procedure.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

Requested by a Maintainer, seconded by a Seniormin, more logging is good.
Closes issue #9843 
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

![Organ Manipulation 2](https://github.com/BeeStation/BeeStation-Hornet/assets/97719613/a8a82ecc-5139-493d-93d3-a34ba91aab12)
![Cavity Implant 2](https://github.com/BeeStation/BeeStation-Hornet/assets/97719613/a4796103-056a-49ed-8369-0506476acf55)
![Implant Removal 2](https://github.com/BeeStation/BeeStation-Hornet/assets/97719613/50b035e9-487e-44dc-b6d4-5bc8c600d703)

</details>

## Changelog
:cl: Impish_Delights
admin: additional logging for organ manipulation, cavity implant, and implant removal surgeries
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

VickiMorris committed:
Logging for Inserting & Removing Surgically

Adds logging for inserting new organs via Organ Manipulation.
Adds logging for extracting implants from people.
Adds logging for storing or removing items surgically using a cavity implant.

EvilDragonfiend reviewed (changes requested):
Would please you add logging when they 'attempt' an operation aside finished operation?

VickiMorris committed:
Logs Surgery Attempts, Removes Surgery Intent

Adds an additional log for when someone attempts to implant/remove organs, implant/remove cavity items, and remove/check for implants.
Removed user intent for surgical steps, as they can only be done on help, or disarm (Intentional Failure) which causes a combat log anyway.

EvilDragonfiend reviewed (approved):
looks good to me
